akka-docs/src/main/paradox/coexisting.md

```diff
@@ -0,0 +1,93 @@
+# Coexistence
+
+We believe Akka Typed will be adopted in existing systems gradually and therefore it's important to be able to use typed 
+and untyped actors together, within the same `ActorSystem`. Also, we will not be able to integrate with all existing modules in one big bang release and that is another reason for why these two ways of writing actors must be able to coexist.
+
+There are two different `ActorSystem`s: `akka.actor.ActorSystem` and `akka.actor.typed.ActorSystem`. 
```

Comment:
not true any more

Reply:
From the user perspective I think this still holds, there are two different classes with different APIs

Reply:
right, not sure what I was thinking. In this coexisting section it is good to show how to start out with untyped.ActorSystem and convert. In other examples I assume we will show starting `typed.ActorSystem`.

Perhaps clarify that `typed.ActorSystem` starts an untyped.ActorSystem under the hood so that there are no surprises from the "leaky" abstraction layers

Reply:
I'll add a comment to explain and an example of converting a untyped to a typed actor system